projects/core/src/error-handling/effects-error-handler/cx-error-handler.effect.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, inject } from '@angular/core';
+import { Actions, createEffect } from '@ngrx/effects';
+import { Observable } from 'rxjs';
+import { filter, tap } from 'rxjs/operators';
+import { FeatureConfigService } from '../../features-config';
+import { ErrorAction } from '../../model/index';
+import { ErrorActionService } from './error-action.service';
+
+/**
+ * Effect that captures in a centralized manner errors occurred in NgRx flow.
+ * When such an action is detected, it delegates the error handling to the `EffectsErrorHandlerService`.
```

Comment:
[MINOR]
Outdated docs.
```suggestion
 * When such an action is detected, it delegates the error handling to the `ErrorActionService`.
```